### DISCUSSION TITLE
n8n и почта

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#312] [Как это сделать?] [answered]
В инструкции по отправке сообщения из totum в Slack через n8n указано как отправить, но жаль что нет примера как получать сообщения откуда-то в Тотум.

Задача состоит в том, чтобы новые письма складывать в тотум.
Т.е. в тотуме делаем таблицу Письма. При получении сообщения imap из n8n будет создаваться строка и сообщение должно разбиваться по полям: контакт, сообщение, вложения и т.п.

Может быть у кого-то есть опыт как это сделать? Подскажите пожалуйста, как правильно настроить связь IMAP trigger и totum, не пойму как их соединить.
![Screenshot_11](https://user-images.githubusercontent.com/113771444/212292543-84dde216-21ab-4e36-a8aa-f6693c790e9e.jpg)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Что-то сюда никто не дошел. Чуть позже в этом году напишем чуть больше инструкций по n8n (но это не точно)
